benchmark/all-robots.cpp

```diff
@@ -153,35 +197,8 @@ void print_benchmark(RobotEENames robot) {
               << " (max: " << duration.maxCoeff() << ", min: " << duration.minCoeff()
               << ", per nodes: " << avg[ithread] * (ithread + 1) / N << " +- " << stddev[ithread] * (ithread + 1) / N
               << ")" << std::endl;
-  }
-
-  /*******************************************************************************/
-  /************************* DIFFERENTIAL ACTION TIMINGS *************************/
-  for (int ithread = 0; ithread < NUM_THREADS; ++ithread) {
-    duration.setZero();
-#ifdef WITH_MULTITHREADING
-    omp_set_num_threads(ithread + 1);
-#endif
-    for (unsigned int i = 0; i < T; ++i) {
-      crocoddyl::Timer timer;
-#ifdef WITH_MULTITHREADING
-#pragma omp parallel for
-#endif
-      for (unsigned int j = 0; j < N; ++j) {
-        boost::shared_ptr<crocoddyl::IntegratedActionModelEuler> m =
-            boost::static_pointer_cast<crocoddyl::IntegratedActionModelEuler>(problem->get_runningModels()[j]);
-        boost::shared_ptr<crocoddyl::IntegratedActionDataEuler> d =
-            boost::static_pointer_cast<crocoddyl::IntegratedActionDataEuler>(problem->get_runningDatas()[j]);
-        m->get_differential()->calc(d->differential, xs[j], us[j]);
-      }
-      duration[i] = timer.get_us_duration();
-    }
-    avg[ithread] = AVG(duration);
-    stddev[ithread] = STDDEV(duration);
-    std::cout << ithread + 1 << " threaded diff calc [us]: \t" << avg[ithread] << " +- " << stddev[ithread]
-              << " (max: " << duration.maxCoeff() << ", min: " << duration.minCoeff()
-              << ", per nodes: " << avg[ithread] * (ithread + 1) / N << " +- " << stddev[ithread] * (ithread + 1) / N
-              << ")" << std::endl;
+    csv << "calc" << (ithread + 1) << false << avg[ithread] << stddev[ithread] << duration.maxCoeff()
```

Comment:
Is this printing an output in the terminal as well?
It would be good if it does so.

Reply:
We already have a readable terminal output. Polluting that with an un-readable and duplicate stream is not a good idea

Reply:
I see it now. Thanks for the explanation